Tabloid/client/src/components/comment/PostComment.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext, useEffect } from "react";
+import { CommentContext } from "../../providers/CommentProvider";
+import { Container, Table } from "reactstrap";
+import Comment from "./Comment";
+import { Link } from "react-router-dom";
+import { useHistory, useParams } from "react-router-dom";
+
+const CommentList = () => {
+const { comments, getCommentsByPostId } = useContext(CommentContext);
+const { id } = useParams();
+useEffect(() => {
+    getCommentsByPostId(id);
+}, []);
+
+return (
+    <>
+    {comments.length > 0 ? 
+        <>
+        <Container className="justify-content-center" fluid={true}>
+                {comments.map((comment, index) => {
```

Comment:
Here you can add the title of the post before the comments map begins and add a button that redirects back to the post details page

Reply:
Done!